tests/queries/0_stateless/03669_into_outfile_create_parent_directories.sh

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env bash
+
+CUR_DIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+. "$CUR_DIR"/../shell_config.sh
+
+BASE_DIR=/tmp/clickhouse_test_into_outfile_create_parent_directories_setting
+CLIENT_OUTDIR="${BASE_DIR}/client"
+LOCAL_OUTDIR="${BASE_DIR}/local"
```

Comment:
Sometimes we execute the test in parallel with itself (to check its flakiness), so it's better to use unique folders names. You can use `$CLICKHOUSE_TEST_UNIQUE_NAME` for it. Also, let's create these directories in the current directory:
```suggestion
BASE_DIR="${CUR_DIR}/${CLICKHOUSE_TEST_UNIQUE_NAME}"
CLIENT_OUTDIR="${BASE_DIR}/client"
LOCAL_OUTDIR="${BASE_DIR}/local"
```